### ISSUE TITLE
Site is unavailable over SSL

### ISSUE THREAD

**Issue by QuLogic** [scipy/SciPyCentral#204]
When loading https://central.scipy.org/, you get an error message: SSL peer has no certificate for the requested DNS name. (Error code: ssl_error_unrecognized_name_alert) which seems to indicate that the server is running SNI without a certificate for this domain.

**Comment by ksurya** [comment]
Yes. We do not yet support HTTPS connection. We shall take this is enhancement request.
